app/src/main/java/com/project/pv239/customtimealarm/activities/WakeUpActivity.java

```diff
@@ -124,6 +122,36 @@ public void onClick(View v) {
                 }
             }
     }
+
+    @Override
+    public void onBackPressed() {
+    }
+
+    public static void acquireLock(Context context) {
+        PowerManager pm = (PowerManager)  context.getSystemService(Context.POWER_SERVICE);
+        if (pm != null) {
+            sWakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "alarm lock");
+            sWakeLock.acquire(10000);
```

Comment:
co toto robi pls? :)

Reply:
kopiroval som ten kod ale malo by to ziskat wake lock na 10 sekund (aby sa nezasekol ked to padne nejako) a uvolni sa ked sa spusti aktivita

Reply:
uuu nice